frontend/src/components/form/StepperNavigation.vue

```diff
@@ -39,20 +39,20 @@ function handlePrevClick() {
 <template>
   <div class="flex pt-4 justify-content-between">
     <Button
+      :style="{ visibility: props.prevDisabled ? 'hidden' : 'visible' }"
```

Comment:
Preferred to not use style prop. Instead use create a class with the CSS attributes and dynamically insert the classes there.